cluster/saltbase/salt/kube-apiserver/default

```diff
@@ -44,6 +44,12 @@
 {% set token_auth_file = "--token_auth_file=/dev/null" -%}
 
 {% if grains.cloud is defined -%}
+{% if grains.cloud in [ 'gce' ] -%}
```

Comment:
Optimism. I can make it a straight == if you'd prefer until we have other cloud providers working.